packages/core/discovery.config.default.js

```diff
@@ -100,5 +100,6 @@ module.exports = {
     cypressVersion: 12,
     enableCypressExtension: false,
     noRobots: false,
+    preact: false,
```

Comment:
```suggestion
    enablePreact: false,
```

question: should we follow this pattern?

Reply:
I don't know (thinking...), because when we add **preact** we are replacing **react**. Either option is ok for me.

Reply:
I believe this needs to be documented.

Reply:
I don't have a preference in this case, but I agree that it should be documented like the [`enableCypressExtension`](https://developers.vtex.com/docs/guides/faststore/project-structure-custom-cypress) is!

Reply:
I will merge this PR, okay?